codalab/lib/worksheet_util.py

```diff
@@ -683,6 +683,10 @@ def flush_bundles():
         if len(bundle_infos) == 0:
             return
 
+        def raise_usage_error():
```

Comment:
This method was meant for only certain type of usage errors involving
display mode genpath
If you move it up here, then rename to raise_genpath_usage_error()